cdc/log.cc

```diff
@@ -1284,15 +1295,17 @@ class transformer final {
                         cdc_op = marker.is_live() ? operation::insert : operation::update;
 
                         if (ttl) {
-                            set_ttl(log_ck, *ttl, res);
+                            set_ttl(log_ck, *ttl);
                         }
                     }
-                    set_operation(log_ck, cdc_op, res);
+                    set_operation(log_ck, cdc_op);
                 }
             }
         }
+    }
 
-        return res;
+    std::vector<mutation> take_generated_log_mutations() {
```

Comment:
Maybe take_generated_log_mutations() && { to make it clear that the function destroys the object?

Reply:
Done